docs/src/paradox/01-introduction/what-is-knora.md

```diff
@@ -19,51 +19,101 @@ License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+sense to them, while allowing Knora to support searching and linking across projects.
+
+Knora has built-in support for data structures that are commonly needed in
+humanities data, and that present unique challenges for any type of database storage.
+
+### Calendar-Independent Dates
+
+In the humanities, a date could be based on any sort of calendar (e.g.
+Gregorian, Julian, Islamic, or Hebrew). Knora stores dates using a calendar-independent,
+astronomical representation, and converts between calendars as needed. This makes
+it possible to search for a date in one calendar, and get search results in other calendars.
+
+### Flexible, Searchable Text Markup
+
+Commonly used text markup systems, such as [TEI/XML](http://www.tei-c.org/),
+have to represent a text as a hierarchy, and therefore have trouble supporting
+overlapping markup. Knora supports @ref:[Standoff/RDF markup](standoff-rdf.md): the markup is stored
+as RDF data, separately from the text, allowing for overlapping markup. Knora's RDF-based standoff
+is designed to support the needs of complex digital critical editions. Knora
+can import any XML document (including TEI/XML) for storage as standoff/RDF,
+and can regenerate the original XML document at any time.
+
+## Powerful Searches
+
+Knora's API provides a search language, @ref:[Gravsearch](../03-apis/api-v2/query-language.md),
+that is designed to meet the needs of humanities researchers. Gravsearch supports Knora's
+humanites-focused data structures, including calendar-independent dates and standoff markup, as well
+as fast full-text searches. This allows searches to combine text-related criteria with any other
+criteria. For example, you could search for a text that contains a certain word
+and also mentions a person who lived in the same city as another person who is the
+author of a text that mentions an event that occurred during a certain time period.
+
+## Access Control
+
+The RDF standards do not include any concept of permissions. Knora's permission
+system allows project administrators and users to determine who can see or
+modify each item of data. Knora filters search results according to each
+user's permissions.
+
+## Data History
+
+RDF does not have a concept of data history. Knora maintains all previous
+versions of each item of data. Ordinary searches return only the latest version,
+but it is possible to obtain an item as it was at any point in the past.
+
+## Data Consistency
+
+RDF triplestores do not implement a standardised way of ensuring the consistency
+of data in a repository. Knora ensures that all data is consistent, conforms
+the project-specific data models, and meets Knora's minimum requirements
+for interoperability and reusability of data.
+
+## Linked Open Data
+
+Knora supports publishing data online as as [Linked Open Data](http://linkeddata.org/),
+using open standards to allow interoperability between different repositories
+on the web.
+
+## Build Your Own Application
```

Comment:
This is outdated. You can now build your own VREs using modules provided by KnoraUI (https://github.com/dhlab-basel/Knora-ui). SALSAH is just one possible VRE that make use of Knora UI.